package/dev-requirements.txt

```diff
@@ -0,0 +1,5 @@
+
+--constraint="https://raw.githubusercontent.com/apache/airflow/constraints-2.3.3/constraints-3.7.txt"
+apache-airflow==2.3.3
+
+-e .[test]
```

Comment:
Do we actually need all this to build the package?

Reply:
Yeah. This is for testing. I had issues when installing airflow, using the constraint solved it but I will investigate again